<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDepthDialog</class>
 <widget class="QDialog" name="ProfileDepthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Depth</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="PaintWidget" name="drawing_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="coords_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>d= z=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="orientation_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Orientation : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="export_csv_btn">
         <property name="text">
          <string>Export profile as csv</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_btn">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header location="global">paint_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
